src/main/java/com/notably/logic/parser/NotablyParser.java

```diff
@@ -32,6 +36,7 @@
             + "To see the list of available commands, type: help";
 
     private Model notablyModel;
+    private final Logger logger = LogsCenter.getLogger(getClass());
```

Comment:
Oops sorry missed this before. Just a tiny thing. Let's make this `logger` a `private static final` field since we don't need to create one for each instance of `NotablyParser`!